extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,33 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: MQTT Generic Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.mqtt.generic;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.10.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: 
+ com.google.common.collect,
+ org.apache.commons.lang,
+ org.apache.commons.net.util,
+ org.eclipse.jdt.annotation;resolution:=optional,
+ org.eclipse.smarthome.binding.mqtt.generic,
+ org.eclipse.smarthome.binding.mqtt.generic.handler,
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.config.discovery,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.transform,
+ org.eclipse.smarthome.core.types,
+ org.eclipse.smarthome.io.transport.mqtt,
+ org.osgi.framework,
+ org.osgi.service.component,
+ org.osgi.service.component.annotations;resolution:=optional,
+ org.slf4j
+Service-Component: OSGI-INF/*.xml
+Export-Package: org.eclipse.smarthome.binding.mqtt.generic,
+ org.eclipse.smarthome.binding.mqtt.generic.handler
+Bundle-ActivationPolicy: lazy
```

Comment:
Hm, as it is only necessary for the Eclipse IDE and not pure OSGi, I don't think we need it.
Is there already an agreement, if this should be used or not?